tests/knitr.R

```diff
@@ -1,9 +1,9 @@
-if (suppressPackageStartupMessages(requireNamespace("knitr", quietly = TRUE))) {
-    require(knitr)
-    knit("knitr.Rmd", quiet=TRUE)
-    cat(readLines("knitr.md"), sep="\n")
-    invisible(file.remove("knitr.md"))
-} else {
-    cat(readLines("knitr.Rout.mock", warn = FALSE), sep="\n")
+if (!suppressPackageStartupMessages(requireNamespace("knitr", quietly=TRUE))) {
+  cat(readLines("knitr.Rout.mock", warn=FALSE), sep="\n")
+  q('no')
 }
 
+require(knitr)
+knit("knitr.Rmd", quiet=TRUE)
+cat(readLines("knitr.md"), sep="\n")
+invisible(file.remove("knitr.md"))
```

Comment:
https://github.com/Rdatatable/data.table/blob/master/tests/knitr.Rout.mock
no need to update here?

Reply:
Good point about updating the goldens. I'm not sure how to generate the `.mock` correctly, working on it.